phys/module_pbl_driver.F

```diff
@@ -2203,6 +2213,26 @@ SUBROUTINE pbl_driver(                                          &
               ,ITS=its,ITE=ite,JTS=jts,JTE=jte,KTS=kts,KTE=kte)
           ENDIF
 
+
+     IF (PRESENT(ahe)) THEN
```

Comment:
@epn09 This may not be the best place to add this anthropogenic heating flux. First, if putting it in this pbl driver module, should it be added to a place after land surface model and urban scheme is called? Second, it may be better to include this to the SLUCM code module instead of pbl driver to avoid messing up with other physics.

Reply:
@cenlinhe The reason we didn't add AH flux to the urban scheme is because the construction of the AH dataset did not assume that AH comes from only urban. 

* Energy loss are distributed equally for all grids within a country. 
* There are strong point sources of AH (power station, steel mills) that do not necessarily belong to urban grids.

What's your opinion on this?

The full paper is available here (https://www.nature.com/articles/s41597-021-00850-w).

Reply:
OK, I see. This makes sense to me, as the AH is not only for urban AH. power station may also belong to urban?

Reply:
@cenlinhe Power station may be in either rural or urban. When constructing the AH dataset we did not make that distinction.